src/components/DistanceRequest/index.js

```diff
@@ -134,7 +162,7 @@ function DistanceRequest({transactionID, report, transaction, route, isEditingRe
     }, [waypoints, previousWaypoints]);
 
     const navigateBack = () => {
-        Navigation.goBack(isEditing ? ROUTES.MONEY_REQUEST_CONFIRMATION.getRoute(iouType, reportID) : ROUTES.HOME);
+        Navigation.goBack(isEditingNewRequest ? ROUTES.MONEY_REQUEST_CONFIRMATION.getRoute(iouType, reportID) : ROUTES.HOME);
```

Comment:
Shouldn't this be `(isEditing || isEditingNewRequest)`?

Reply:
Oh I see right, when editing a request, and then going back, we just close the modal.